tests/python/tests/test_spark.py

```diff
@@ -61,6 +61,15 @@ def test_replace_table_pyspark(spark, warehouse: conftest.Warehouse):
     sdf.writeTo(f"test_replace_table_pyspark.my_table").createOrReplace()
 
 
+def test_create_view(spark, warehouse: conftest.Warehouse):
```

Comment:
Could we add some more tests regarding properties, dropping views and altering them (see https://iceberg.apache.org/docs/1.5.1/spark-ddl/#iceberg-views-in-spark), we can also add them in another PR